src/App.tsx

```diff
@@ -8,32 +9,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  const onTabSelected = (value: string) => {
+    setSelectedTabId(value);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is&nbsp;Tab 1
+        {`Selected tab is ${selectedTab?.title}`}
       </h1>
 
       <div>
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active">
-              <a href="#tab-1">Tab 1</a>
-            </li>
-
-            <li className="">
-              <a href="#tab-2">Tab 2</a>
-            </li>
-
-            <li className="">
-              <a href="#tab-3">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="tab-content">
-          Some text 1
-        </div>
+        <Tabs
+          tabs={tabs}
+          Select={onTabSelected}
+          selectedTabId={selectedTabId}
+          selectedTab={selectedTab}
```

Comment:
why do you need both of these props and variables, I guess you should have only one of them